### ISSUE TITLE
Android crash at Startup

### ISSUE THREAD

**Issue by lordfiSh** [nextcloud/android#6574]
```
************ CAUSE OF ERROR ************

java.lang.RuntimeException: Unable to resume activity {com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.endsWith(java.lang.String)' on a null object reference
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4246)
	at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:4278)
	at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:52)
	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2040)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:224)
	at android.app.ActivityThread.main(ActivityThread.java:7520)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:539)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.endsWith(java.lang.String)' on a null object reference
	at com.owncloud.android.datamodel.OCFile.getDecryptedRemotePath(OCFile.java:212)
	at com.owncloud.android.datamodel.OCFile.getDecryptedFileName(OCFile.java:388)
	at com.owncloud.android.datamodel.OCFile.getFileName(OCFile.java:379)
	at third_parties.daveKoeller.AlphanumComparator.compare(AlphanumComparator.java:91)
	at com.owncloud.android.utils.FileSortOrderByName.lambda$sortCloudFiles$0(FileSortOrderByName.java:54)
	at com.owncloud.android.utils.-$$Lambda$FileSortOrderByName$YdISYWUaQVK-4PIrs4OOhoEHgbQ.compare(Unknown Source:6)
	at java.util.TimSort.countRunAndMakeAscending(TimSort.java:360)
	at java.util.TimSort.sort(TimSort.java:220)
	at java.util.Arrays.sort(Arrays.java:1492)
	at java.util.ArrayList.sort(ArrayList.java:1470)
	at java.util.Collections.sort(Collections.java:206)
	at com.owncloud.android.utils.FileSortOrderByName.sortCloudFiles(FileSortOrderByName.java:52)
	at com.owncloud.android.ui.adapter.OCFileListAdapter.swapDirectory(OCFileListAdapter.java:912)
	at com.owncloud.android.ui.fragment.OCFileListFragment.listDirectory(OCFileListFragment.java:1261)
	at com.owncloud.android.ui.fragment.OCFileListFragment.listDirectory(OCFileListFragment.java:1214)
	at com.owncloud.android.ui.fragment.OCFileListFragment.listDirectory(OCFileListFragment.java:1203)
	at com.owncloud.android.ui.activity.FileDisplayActivity.updateListOfFilesFragment(FileDisplayActivity.java:629)
	at com.owncloud.android.ui.activity.FileDisplayActivity.onResume(FileDisplayActivity.java:1103)
	at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1454)
	at android.app.Activity.performResume(Activity.java:8050)
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4236)
	... 11 more

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30130051
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: Xiaomi
Device: polaris
Model: Mi MIX 2S
Id: QKQ1.190828.002
Product: polaris

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: V11.0.5.0.QDGMIXM
```

App asked me to post here ;) 
After that I only get the Fingerprint/Pin Login Screen, but I can't get pass them - stuck in Fingerprint Loop

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky I think this will be fixed by your E2EE improvements (3.13.0 RC2), right?

**Comment by AndyScherzinger** [comment]
#6557 should fix this, please update to RC2 and re-open if issue persists
